frontend/src/components/studio/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'common/types/types';
+import { Icon, Link } from 'frontend/src/components/common/common';
+import { IconName, IconColor, AppRoute } from 'common/enums/enums';
+import cn from 'clsx';
+import { ISideBarItem, sideBarItems } from './config';
+import { useLocation } from 'hooks/hooks';
+
+import styles from './styles.module.scss';
+
+const StudioSidebar: FC = () => {
+  const { pathname } = useLocation();
+  const getColor = (item: string): string => (pathname === item ? IconColor.GREEN : IconColor.WHITE);
```

Comment:
Not understandable enough. Better to do it in thie way:

```
const isActiveTab = ...

// then add the color using isActiveTab var 
```

Reply:
In addition, your solution will not work when you have bigger route name. Like:

`/home/route-name/route-name`